feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go

```diff
@@ -0,0 +1,296 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package setup is scoped only to be used for scripts in path
+// feature/experimental/system/gnmi/benchmarking/ate_tests/
+// Do not use elsewhere.
+package setup
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	ISISInstance   = "DEFAULT"        // ISIS instance name.
+	ISISMetric     = 100              // ISIS metric.
+	DUTAreaAddress = "49.0001"        // DUT ISIS area address.
+	DUTSysID       = "1920.0000.2001" // DUT ISIS system ID.
+	RouteCount     = 200              // Route count.
+	PeerGrpName    = "BGP-PEER-GROUP" // BGP peer group name.
+	DUTAs          = 64500            // DUT AS.
+	ATEAs          = 64501            // ATE AS.
+
+	ateAs2                = 64502
+	dutStartIPAddr        = "192.0.2.1"
+	ateStartIPAddr        = "192.0.2.2"
+	plenIPv4              = 30
+	advertiseBGPRoutesv4  = "203.0.113.1"
+	advertiseISISRoutesv4 = "198.18.0.0"
+	authPassword          = "ISISAuthPassword"
+)
+
+var (
+	DUTIPList       = make(map[string]net.IP) // DUT IP list.
+	ATEIPList       = make(map[string]net.IP) // ATE IP list.
+	ISISMetricArray []uint32                  // ISIS metric array.
+	ISISSetBitArray []bool                    // ISIS set bit array.
+)
+
+// BuildIPList builds list of ip addresses for the ports in binding file.
+// (Both DUT and ATE ports).
+func BuildIPList(t *testing.T) {
```

Comment:
This method can accept (dut *ondatra.DUTDevice) instead of testing.T

Reply:
Done.